src/main/java/com/amazonaws/encryptionsdk/jce/JceMasterKey.java

```diff
@@ -234,12 +237,17 @@ public WrappingData(final Cipher cipher, final byte[] extraInfo) {
     }
 
     private static class Rsa extends JceMasterKey {
+        private static final Pattern SUPPORTED_TRANSFORMATIONS =
```

Comment:
It might be nice to have some kind of test coverage for this in general in the future to make sure we warn / don't warn as intended